### PR TITLE
Login UI markup

### PR DESCRIPTION
**Pull request by leonplondon**
# Summary
Implemented UI for login feature.

# Trello Card
https://trello.com/c/Sdk0UL8J/16-u1-frontend-login-componentes-b%C3%A1sicos

# Screenshots
screenshot_1542920689

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
+        android:name=".example.NewsApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:theme="@style/AppTheme">
 
-        <activity
-            android:name=".example.ui.example.ExampleActivity"
-            android:screenOrientation="portrait">
+        <!-- <activity -->
```
Comment (DemianIac) [inline app/src/main/AndroidManifest.xml]: We can remove this commented code or add a //TODO to remove later.

File: app/src/main/res/layout/fragment_login.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:adjustViewBounds="true"
+        android:scaleType="centerCrop"
+        android:src="@drawable/img_login_cover"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <View
```
Comment (DemianIac) [inline app/src/main/res/layout/fragment_login.xml:19]: width should be match_partent here. Try to never use two properties  (android:layout_width and and  android:layout_height) with size 0dp at the same time.

**Review by DemianIac — CHANGES_REQUESTED** [review]
This is an awesome work Darío . Just two minor changes.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (leonplondon) [reply app/src/main/AndroidManifest.xml]: I think is better to remove them

File: app/src/main/res/layout/fragment_login.xml (hunk 2 above)
Comment (leonplondon) [reply app/src/main/res/layout/fragment_login.xml:19]: Changed attribute to be match_parent

**Pull request merged**